web-app/src/localplay/games/localplay.games.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Body, Controller, Post, Put } from '@nestjs/common';
+import { GameState, initNewGameState, updateGameState } from '../../common/gameState';
+import { Coordinates } from '../../common/utils';
+
+@Controller('localplay/games')
```

Comment:
What about adding an "api" prefix into rest api endpoint for non ssr controllers ?

Reply:
good idea 🙂